src/core/biblio.js

```diff
@@ -63,7 +63,7 @@ const endNormalizer = function(endStr) {
 
 const endWithDot = endNormalizer(".");
 
-export function wireReference(rawRef, target = "_blank") {
+export function wireReference(rawRef, target = "_blank",style) {
```

Comment:
Sorry! I was wrong here. `stringifyReference` below was correct.

Reply:
Oh alright! So I should remove that argument from there then. I also realized I have to add " " for title in MLA. Should I be doing anything else differently?